mailer/tests.py

```diff
@@ -595,29 +598,29 @@ def test_message_log(self):
 
     def test_message_str(self):
         with self.settings(MAILER_EMAIL_BACKEND="django.core.mail.backends.locmem.EmailBackend"):
-            mailer.send_mail("Subject Msg", "Body", "msg1@example.com", ["rec1@example.com"])
+            mailer.send_mail("Subject Msg 中", "Body 中", "msg1@example.com", ["rec1@example.com"])
 
             self.assertEqual(Message.objects.count(), 1)
 
             msg = Message.objects.get()
             self.assertEqual(
                 str(msg),
-                'On {0}, \"Subject Msg\" to rec1@example.com'.format(msg.when_added),
```

Comment:
The other way to expose this problem would be an integration test but that's more work to setup. Let me know that's the route you'd prefer me to take for testing this.